js/index.js

```diff
@@ -0,0 +1,181 @@
+var Bitcore = require('bitcore-lib-dash');
+
+var socket;
+var paymentCycle;
+
+var mainnetProvider = 'https://insight.dash.org:443';
+var mainnetPrefix = '/api';
+
+var testnetProvider = 'https://test-insight.dash.org:443';
+var testnetPrefix = '/api';
+
+var init = function(network, provider, prefix) {
+    var gov = new Bitcore.GovObject.Proposal();
+    gov.network = network || 'mainnet';
+
+    paymentCycle = new PaymentCycle(gov, provider, prefix);
+
+    socket = io(provider);
+
+    socket.on('connect', function() {
+        socket.emit('subscribe', 'inv');
+        console.log("socket.io initialized...");
+    });
+
+    socket.on('disconnect', function() {
+        console.log('connection lost');
+        alert('We sorry; connection lost.');
```

Comment:
We want to be really sure you know the connection has been lost :p 

I understand the console.log, but an alert is maybe unnecessary in my opinion. 
I would prefer a UI things notifying that, or the button being blocked or something else